docs: storage cleanup (#16785)
supabase/supabase: apps/docs/pages/guides/storage.mdx

  id: 'storage',
  title: 'Storage',
  description: 'Use Supabase to store and serve files.',
  subtitle: 'Use Supabase to store and serve files.',
  sidebar_label: 'Overview',
}

> Supabase Storage makes it simple to store and serve large files, providing a robust framework for file access controls.

Reviewer: ```suggestion
Supabase Storage makes it simple to upload and serve files of any size, providing a robust framework for file access controls.
```

## Features

> You can use Supabase Storage to store images, videos, documents, and any other file type. It provides includes a global CDN to reduce latency, serving your files from over 285 cities globally. Supabase Storage includes a built-in image resizer, so you can resize and compress your media files on the fly.

Reviewer: ```suggestion
You can use Supabase Storage to store images, videos, documents, and any other file type. Serves your assets with a global CDN to reduce latency from over 285 cities globally. Supabase Storage includes a built-in image optimizer, so you can resize and compress your media files on the fly.
```

## Examples

> Check out all of the AI [templates and examples](https://github.com/supabase/supabase/tree/master/examples/storage) in our GitHub repository.

Reviewer: ```suggestion
Check out all of the Storage [templates and examples](https://github.com/supabase/supabase/tree/master/examples/storage) in our GitHub repository.
```
